homeassistant/generated/integrations.json

```diff
@@ -6166,6 +6166,12 @@
           "config_flow": false,
           "iot_class": "local_polling",
           "name": "TP-Link LTE"
+        },
+        "tplink_tapo": {
+          "integration_type": "virtual",
+          "config_flow": false,
```

Comment:
Hassfest generated this, even when a similar manifest.json for symfonisk has this set to true :thinking:

Reply:
Seems to work anyways

Reply:
Nvm, I was reading a wrong entry.